README.md

```diff
@@ -1 +1,188 @@
-# RoR_budget_App
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budgets App <a name="about-project"></a>
+
+**Budgets** is a mobile web application that allows you to manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+# This project was built with
+
+- Ruby on Rails
+
+- PostgreSQL
+
+- Bootstrap CSS Framework
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Aunthantication**
+- **Add Categories**
+- **Add Transaction**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+----
+_[Here](https://www.loom.com/share/580d903c46f646d18b7afdebb1180fce) is the link to the project presentation video_
+
+<!-- _[Here](https://budget-app-nqso.onrender.com/) is the link to the live demo_ -->
+
```

Comment:
- [ ] Great job working on the requested changes but I think it would be great if you deploy your project as mentioned by the previous reviewer since you commented here the link so kindly add to be deployed online 🙏🏻🔥♻💪🏻